### PR TITLE
Removes a lavaland_surface_ww_vault.dmm because its just bad.

### PR DESCRIPTION
**Pull request by Jalleo**
:cl: Jalleo
del: WW_maze from lavaland has been removed it wasnt that good really anyhow
/:cl:

Cheridan Jalleo: i'm not even sure which one the wildwest ruin is

Cheridan is it newer?
Jalleo it is the one from the away mission Cheridan
Jalleo let me get a pic in a minuite
Jalleo its not good at all really
Cheridan oh the one thats just the big maze
Jalleo yeah
Jalleo do you want that one removed?
QualityVan the one full of the fallen?
coiax the big maze with the free syndihardsuit
Cheridan had to think because it ends up having nothing to do with the wild west at all 
QualityVan didn't they mean westworld
Cheridan it's not particularly interesting
Cheridan it's just a huge maze of mobs, if you're looking for ruins to nix i'd go for it
Cheridan the meat grinder gimmick where someone has to sacrifice themselves to get to the endis interesting but i think you can bypass it

### CODE REVIEW

**Comment by coiax** [comment]
This uh... PR doesn't remove anything...

**Comment by Jalleo** [comment]
Fuck

**Comment by Jalleo** [comment]
Give me a minuite and then it is fixed

**Comment by Jalleo** [comment]
Uh why did Travis cancel itself?

**Comment by Repukan** [comment]
Why don't you remove the sloth ruin instead.

That one is even more useless

**Comment by FantasticFwoosh** [comment]
Assassinate both the maze ruins, the surprise is lost when you first navigate the whole thing.

**Comment by Jalleo** [comment]
If @Cheridan wants those removed too I shall remove them. I dont want a maintainer to try and prevent the bad ones from getting removed.

**Comment by WJohn** [comment]
For reference: This thing was GIGANTIC, literally taking up half of lavaland horizontally, and a huge eyesore at that.

**Comment by KorPhaeron** [comment]
You need to make something to replace it if you want to remove it. Having bad stuff in the pool is better than having a tiny pool of the same things every round

**Comment by coiax** [comment]
Removing a big ruin like this will make golems more likely though. Everyone loves golems.

**Comment by KorPhaeron** [comment]
It will also make morphs more likely though and I dont want to hurt Oldman again so soon

**Comment by Repukan** [comment]
Oh yes let's make ghost spawns abusable for double lives more common.

**Comment by ChangelingRain** [comment]
Removing this will make the hierophant boss's arena, and the swarmer boss if it's not disabled(?), more likely to spawn.

**Comment by Repukan** [comment]
What about that vendor ruin.

The one that was a square with a bunch of vendors

**Comment by coiax** [comment]
We do not keep bad ruins in because otherwise other things may appear too much. If that's really an issue, lower the ruin budget on lavaland.

**Comment by KorPhaeron** [comment]
@Repukan

I think someone already removed that

**Comment by Jalleo** [comment]
I am going to throw down a idea thread for ruins in general tomorrow. Really want just need to completly go mad with them or not at all. 
Leaving in poorly designed ones until then is a awful duct tape fix.

**Comment by WJohn** [comment]
Yeah the excuse "you need to replace it with something if you're removing it" is not a good enough excuse to keep this piece of shit around. It actively detracts from the quality of a miner's round when he has to spend a huge amount of time walking around this gigantic eyesore, or die fighting to 60+ faithless inside.

And if you're worried about making the heirophant or some other ruins more likely to spawn, why not add an actual CHANCE to spawn something, instead of relying on a shitty price system which holds no preference on spawning a dozen cheap ruins that should be common or only two expensive ones that should be rare?

**Comment by Repukan** [comment]
Nobody is making the miner fight the faithless though?

And has anyone complained about having to path around it? Usually miners follow beelines or use explosive mods to mine 3 rows at a time.

**Comment by ChangelingRain** [comment]
>And if you're worried about making the heirophant or some other ruins more likely to spawn

Oh, no, that's not a worry, that's a bonus, because the hierophant is something I want to show up more.

**Comment by Iamgoofball** [comment]
I would rather get hierophant or swarmers more than this

**Comment by KorPhaeron** [comment]
@wjohn

Well then you have to rewrite the spawn system then. The fact is the whole thing will break down right now if we removed all the mediocre filler ruins. It isn't an ideal situation but it is the one we are in.

**Comment by Jalleo** [comment]
That is why I am only removing the worst one. I will try and make some ruins after reworking how cryotubes and freezers connect to atmos pipes this week.

**Comment by coiax** [comment]
I'm sorry about the ruin system, I can rewrite it to be more percentage based if you're really unhappy with it.

The previous system was just "pick 5 templates" and i wanted to make a system where you could have small ruins that wouldn't stop big ruins from spawning

File: tgstation.dme [hunk 1]
```diff
@@ -657,8 +657,8 @@
 #include "code\game\objects\effects\mines.dm"
 #include "code\game\objects\effects\misc.dm"
 #include "code\game\objects\effects\overlays.dm"
-#include "code\game\objects\effects\proximity.dm"
```
Comment (kyrahabattoir) [inline tgstation.dme]: Wut?

File: tgstation.dme [hunk 2]
```diff
@@ -657,8 +657,8 @@
 #include "code\game\objects\effects\mines.dm"
 #include "code\game\objects\effects\misc.dm"
 #include "code\game\objects\effects\overlays.dm"
-#include "code\game\objects\effects\proximity.dm"
 #include "code\game\objects\effects\portals.dm"
+#include "code\game\objects\effects\proximity.dm"
```
Comment (kyrahabattoir) [inline tgstation.dme]: Wut?

Comment (coiax) [inline tgstation.dme]: Someone didn't put the tgstation.dme in alphabetical order.

**Comment by ChangelingRain** [comment]
This is one of the worst ruins. It is just not worth keeping in even AS filler, and because of how huge it is and how much it costs to spawn, it makes a lot of other, more interesting, ruins way less likely to spawn.

Also isn't one of the the rewards for completing it basically just "you're a shadowperson antag now"

**Pull request merged**